### ISSUE TITLE
Resolve a11y issues with demo env

### ISSUE THREAD

**Issue by hellobrian** [carbon-design-system/carbon#245]
## Detailed description

In order to have an easier time discerning demo vs actual a11y issues, demo a11y should be fixed so it doesn't get picked up by aXe.

> Is this issue related to a specific component? 

No, it's for demo environment

> What did you expect to happen? What happened instead? What would you like to see changed?

I would expect demo environment (`npm run dev`) to show no a11y violations when running aXe

> What browser are you working in? 

Chrome

## Additional information

Previously opened in enterprise: https://github.ibm.com/Bluemix/carbon-issues/issues/275
